command/secrets/secrets_generate.go

```diff
@@ -39,15 +39,6 @@ func (s *SecretsGenerate) DefineFlags() {
 		FlagOptional:      false,
 	}
 
-	s.FlagMap["token"] = helper.FlagDescriptor{
```

Comment:
We are no longer saving a root token to our secrets config, so removing `token` flag from the SDK cli...